### PR TITLE
Add test for -nameout output

### PR DESCRIPTION
**Pull request by richsalz**
Using a cert with Cyrillic characters, kindly supplied by Dmitry Belyavsky

### CODE REVIEW

**Comment by richsalz** [comment]
@beldmit fyi :)

File: test/recipes/25-test_x509.t [hunk 1]
```diff
@@ -15,10 +15,25 @@ use OpenSSL::Test qw/:DEFAULT srctop_file/;
...
 
 require_ok(srctop_file('test','recipes','tconversion.pl'));
 
+my $pem = srctop_file("test/certs", "cyrillic.pem");
+my $out = "cyrillic.out";
+my $msb = srctop_file("test/certs", "cyrillic.msb");
+my $utf = srctop_file("test/certs", "cyrillic.utf8");
+
+ok(run(app(["openssl", "x509", "-text", "-in", $pem, "-out", $out,
+                "-nameopt", "esc_msb"])));
```
Comment (levitte) [inline test/recipes/25-test_x509.t]: Align "-nameopt" with "openssl"

File: test/recipes/25-test_x509.t [hunk 2]
```diff
@@ -15,10 +15,25 @@ use OpenSSL::Test qw/:DEFAULT srctop_file/;
...
+is(cmp_text($out, srctop_file("test/certs", "cyrillic.msb")),
+    0, 'Comparing esc_msb output');
+ok(run(app(["openssl", "x509", "-text", "-in", $pem, "-out", $out,
+                "-nameopt", "utf8"])));
```
Comment (levitte) [inline test/recipes/25-test_x509.t]: Same here

File: test/recipes/25-test_x509.t [hunk 3]
```diff
@@ -15,10 +15,25 @@ use OpenSSL::Test qw/:DEFAULT srctop_file/;
...
 
+my $pem = srctop_file("test/certs", "cyrillic.pem");
+my $out = "cyrillic.out";
+my $msb = srctop_file("test/certs", "cyrillic.msb");
+my $utf = srctop_file("test/certs", "cyrillic.utf8");
+
+ok(run(app(["openssl", "x509", "-text", "-in", $pem, "-out", $out,
+                "-nameopt", "esc_msb"])));
+is(cmp_text($out, srctop_file("test/certs", "cyrillic.msb")),
+    0, 'Comparing esc_msb output');
```
Comment (levitte) [inline test/recipes/25-test_x509.t]: One less space

File: test/recipes/25-test_x509.t [hunk 4]
```diff
@@ -15,10 +15,25 @@ use OpenSSL::Test qw/:DEFAULT srctop_file/;
...
+ok(run(app(["openssl", "x509", "-text", "-in", $pem, "-out", $out,
+                "-nameopt", "utf8"])));
+is(cmp_text($out, srctop_file("test/certs", "cyrillic.utf8")),
+    0, 'Comparing utf8 output');
```
Comment (levitte) [inline test/recipes/25-test_x509.t]: Same here

**Comment by richsalz** [comment]
Updated commit with spacing changes made.

**Comment by richsalz** [comment]
thanks!

**Pull request closed without merging**